scripts/ci/jobs/ocp_qa_e2e_tests.py

```diff
@@ -15,4 +15,8 @@
 os.environ["ROX_RISK_REPROCESSING_INTERVAL"] = "15s"
 os.environ["ROX_SENSOR_CONNECTION_RETRY_MAX_INTERVAL"] = "30s"
 
+if ('aws' in os.environ.get('CLUSTER_PROFILE_NAME', '') or
```

Comment:
Could you please add a comment why/how these env vars relate to the interop tests?

Reply:
> I don't know the context of the workload identities setup and I realize I assumed they are for clusters on gcp and not aws. So I have it checking for the aws cluster type.

The workload identities are only set up for OCP clusters that have been created as `openshift-4` infra flavors. Currently those are created using installer provisioned resources on GCP. So yes today they are only for clusters running on GCP, but that is not a defining property. The workload identities establish a trust between the OpenShift cluster and GCP. In principle you could also have an OpenShift cluster running on AWS, that could use workload identities to authenticate against GCP to use e.g. an artifact registry image integration.

When I added the workload identity setup to this file, I thought that it would only be run on `openshift-4` infra flavors. So ideally we would `SETUP_WORKLOAD_IDENTITIES=true` only for those clusters.